lib/user_agent/browsers.rb

```diff
@@ -5,6 +5,7 @@
 
 require 'user_agent/browsers/acast'
 require 'user_agent/browsers/airr'
+require 'user_agent/browsers/airtraffic_framework'
```

Comment:
This should be named `air_traffic_framework` so it matches up with the correct Pascal casing.

Also, if you plan on working in this gem some more, let's chat about making these requires into autoloads :)

Reply:
Would definitely like to learn about autoload!

Reply:
renamed instances of `airtraffic_framework*` filenames into `air_traffic_framework`